CruzHacks/cruzhacks-2021-website
# navigation bar

Problem
=======
As a prospective hacker,
I would like to view resources in a navigation bar
To more easily navigate the hackathon homepage.

[Link to Asana Ticket](https://app.asana.com/0/1197738625416321/1198219556358101/f)

Solution
========
The navigation bar was implemented previously, but a few things were updated to make this integration more closely represent the provided design. First, the default button styling was updated to appear sleeker and more modern. Then, I changed `buttonProps` to include an enumerated `ButtonType` property to separate primary and secondary buttons (also removing the previous button map from the `Home` page to reduce redundancy; primary buttons are left-aligned and secondary buttons are right-aligned. 

Because a mobile version hasn't been implemented yet, each button just wraps for now, but feedback would be greatly appreciated on how we can scale button sizes more appropriately.

Change summary:
---------------
* first iteration of navigation bar 069b252

Steps to Verify:
----------------
1. Checkout branch using `git checkout feature/navbar`
2. Start local development server using `yarn start`
3. Verify that the navigation bar section is designed correctly and responds accurately.
